plugin/secrets_service_account_key.go

```diff
@@ -170,6 +173,12 @@ func (b *backend) getSecretKey(ctx context.Context, s logical.Storage, rs *RoleS
 	if err != nil {
 		return nil, errwrap.Wrapf("could not read backend config: {{err}}", err)
 	}
+	if cfg == nil {
+		cfg = &config{
```

Comment:
You can just initialize an empty struct here.  No need to query the system values.

Reply:
I missed this panic on first pass and that was the reason for my initial comment.

Reply:
But, you still don't need to set the values :-)